java/squeek/appleskin/ModConfig.java

```diff
@@ -19,6 +19,10 @@ public static void init()
 	{
 		AutoConfig.register(ModConfig.class, JanksonConfigSerializer::new);
 		INSTANCE = AutoConfig.getConfigHolder(ModConfig.class).getConfig();
+
+		// ModConfig must be initialized correctly.
+		if (INSTANCE == null)
+			INSTANCE = new ModConfig();
```

Comment:
Is this actually confirmed to be possible? What happens if this code actually triggers? Would it just continue using the defaults and ignore the user's config?

If it's possible for `AutoConfig.getConfigHolder(ModConfig.class).getConfig()` to return null I think it'd be better to just crash if that happens.

Reply:
for example: config file damaged

you're right, crash might be better